### DISCUSSION TITLE
Questions!!

### DISCUSSION THREAD

**Discussion by gianlore** [Keriew/augustus#455] [Q&A] [answered]
I would like clarification on some things I don't understand:
1) Among the military structures there is "watchtower", what does it mean and what would its function be?
2) Among the options there is "monthly autosave", how does it work?
My English is very lacking, could some charitable spirit clarify my difficulties?
Thanks in advance

**Comment by Areldir** [comment, marked as answer]
Hi there!
1. The Watchtower is a small tower which shoots arrows at the enemy. It is not very strong, but it is cheap, so you can build lots of them to protect your city early in the mission.
2. "Monthly autosave" means that every month in game that passes, the city will be saved. That means if something goes wrong, or the game crashes, you will be able to find a recent save to load.

**Reply by crudelios** [reply]
You can find the autosave in the file name `autosave.svx`.

**Comment by gianlore** [comment]
Ok understood, if I activate the option, at the end of the game I will 
have a file called autosavo.svx, but only one, it is not a MONTHLY save, 
but it happens only once.

------ Messaggio originale ------
Da: "José Cadete" ***@***.***>
A: "Keriew/augustus" ***@***.***>
Cc: "gianlore" ***@***.***>; "Author" 
***@***.***>
Inviato: 17/07/2021 12:39:21
Oggetto: Re: [Keriew/augustus] Questions!! (#455)
